lib/avromatic/model/types/date_type.rb

```diff
@@ -7,11 +7,16 @@ module Model
     module Types
       class DateType < AbstractType
         VALUE_CLASSES = [::Date].freeze
+        INPUT_CLASSES = [::Date, ::Time, ::DateTime].freeze
```

Comment:
That might require a separate PR to the upstream.

Reply:
this is used just for error messaging, but I made it working the same way